docs/guide/config_back_office.md

```diff
@@ -102,13 +102,91 @@ ezpublish:
         <siteaccess>:
             user_preferences:
                 short_datetime_format:
-                    date_format: 'dd/mm/yyy'
+                    date_format: 'dd/mm/yyyy'
                     time_format: 'hh:mm'
                 full_datetime_format:
-                    date_format: 'dd/mm/yyy'
+                    date_format: 'dd/mm/yyyy'
                     time_format: 'hh:mm'
 ```
 
+You can use one of the following filters to format date and time:
+
+- `ez_short_datetime`
+- `ez_short_date`
+- `ez_short_time`
+- `ez_full_datetime`
+- `ez_full_date`
+- `ez_full_time`
+
+The following are examples of using the filters:
+
+``` php hl_lines="3 6"
+<div>
+    // Date formatted in a preferred time zone and short datetime format:
+    {{ content.versionInfo.creationDate|ez_short_datetime }}
+    
+    // Date formatted in UTC and preferred short datetime format:
+    {{ content.versionInfo.creationDate|ez_short_datetime('UTC') }}
+</div>
+```
+
+The filters accept an optional `timezone` parameter for displaying date and time in a chosen time zone.
+The default time zone is set in the User settings menu.
+
+You can also configure date and time format by using the following services:
+
+- `@ezplatform.user.settings.short_datetime_format.formatter`
+- `@ezplatform.user.settings.short_datet_format.formatter`
+- `@ezplatform.user.settings.short_time_format.formatter`
+- `@ezplatform.user.settings.full_datetime_format.formatter`
+- `@ezplatform.user.settings.full_date_format.formatter`
+- `@ezplatform.user.settings.full_time_format.formatter`
+
+To use them, create a `src/AppBundle\Service\MyService.php` file containing:
```

Comment:
Also, maybe move this to Extending eZ Platform? This is not strictly speaking config